### PR TITLE
Update User Authorization doc

### LINKED ISSUE #9463: Update User Authentication Page
Add info on how to create new SQL user, how to change password / delete user (admin only) in the Console

### PR DESCRIPTION
**Pull request by lnhsingh**
- Update headers so it's easier to see what's on the page
- Add instructions on how to manage SQL users in the Console

Closes #9463

### CODE REVIEW

**Review by kannanlakshmi — COMMENTED** [review]
> Reviewed 8 of 9 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9498)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @lnhsingh)

*[a discussion](https://reviewable.io/reviews/cockroachdb/docs/9498#-MRzA0cGDY0_sFD97Xdg:-MRzA0cHDU8EqSVuPvDw:b-o1i3zt) (no related file):*
Everything here LGTM, Lauren, thank you! I will admit that user auth fits better with IAM/Security so if you need another set of eyes, Piyush would be best here but I didn't have any comments as is. 

---

**Comment by lnhsingh** [comment]
TFTR @kannanlakshmi!

@piyush-singh do you mind taking a quick look?

File: cockroachcloud/user-authorization.md [hunk 1]
```diff
@@ -52,71 +50,160 @@ To create a new user, use the [`CREATE USER ... WITH PASSWORD`](../stable/create
...
+On the **SQL Users** page, you can do the following:
+
+- [Create a user](#create-a-sql-user)
+- [Change a user's password](#change-a-users-password)
+- [Remove a user](#remove-a-user)
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="client">
+To list all the users in your cluster, use the [`SHOW USERS`](../stable/show-users.html) statement:
```
Comment (rmloveland) [inline cockroachcloud/user-authorization.md:83]: Based on your recent comments this `/stable/` link may break? Or?

File: cockroachcloud/user-authorization.md [hunk 2]
```diff
@@ -52,71 +50,160 @@ To create a new user, use the [`CREATE USER ... WITH PASSWORD`](../stable/create
...
+
+1. Navigate to the **SQL Users** page.
+1. In the row of the user whose password needs to be changed, click the **...** button.
+
+    <img src="{{ 'images/v20.2/cockroachcloud/sql-users-actions.png' | relative_url }}" alt="Change SQL password" style="border:1px solid #eee;max-width:100%" />
+
+1. From the dropdown, select **Change Password**.
+1. In the **New Password** field, enter the new password.
+
+    The password must be at least 12 characters long.
```
Comment (rmloveland) [inline cockroachcloud/user-authorization.md:114]: Can UI validate/warn on this?  Seems like this is not needed in docs

File: cockroachcloud/user-authorization.md [hunk 3]
```diff
@@ -52,71 +50,160 @@ To create a new user, use the [`CREATE USER ... WITH PASSWORD`](../stable/create
...
+~~~
+
+To assign a user more limited privileges for one table in a database:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> GRANT SELECT, INSERT ON TABLE <database>.<table> TO <user>;
+~~~
+
+For more details, see [Privileges](../v20.2/authorization.html#assign-privileges) and [`GRANT`](../stable/grant.html).
```
Comment (rmloveland) [inline cockroachcloud/user-authorization.md:181]: weird that `/v20.2` is needed for one and `/stable` works for the other?

**Review by rmloveland — APPROVED** [review]
lgtm

**Review by lnhsingh — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9498)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @lnhsingh and @rmloveland)

___
*[cockroachcloud/user-authorization.md, line 83 at r2](https://reviewable.io/reviews/cockroachdb/docs/9498#-MRwPlZqh3-gxYtsgNgz-r2-83:-MSFSPDl--YGzGDEGcm9:bcdjlug) ([raw file](https://github.com/cockroachdb/docs/blob/59a7099b28b27390800ed7468db798e1579ca09c/cockroachcloud/user-authorization.md#L83)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

Based on your recent comments this `/stable/` link may break? Or?
</blockquote></details>

No, it seems for some reason existing mentions of /stable/ work but when I try to add new ones, those break
___
*[cockroachcloud/user-authorization.md, line 114 at r2](https://reviewable.io/reviews/cockroachdb/docs/9498#-MRwPlZqh3-gxYtsgNgz-r2-114:-MSFSZKS-hoCaMn4cELy:bss4a2b) ([raw file](https://github.com/cockroachdb/docs/blob/59a7099b28b27390800ed7468db798e1579ca09c/cockroachcloud/user-authorization.md#L114)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

Can UI validate/warn on this?  Seems like this is not needed in docs
</blockquote></details>

Removed.
___
*[cockroachcloud/user-authorization.md, line 181 at r2](https://reviewable.io/reviews/cockroachdb/docs/9498#-MRwPlZqh3-gxYtsgNgz-r2-181:-MSFSaOY6QVD395NNAyx:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/59a7099b28b27390800ed7468db798e1579ca09c/cockroachcloud/user-authorization.md#L181)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

weird that `/v20.2` is needed for one and `/stable` works for the other?
</blockquote></details>

Done.

**Pull request merged**